-- Found ament_cmake: 1.3.9 (/opt/ros/humble/share/ament_cmake/cmake)
-- Found ament_lint_auto: 0.12.11 (/opt/ros/humble/share/ament_lint_auto/cmake)
Error parsing '/home/cfh/ntest/src/six_dof_arm_moveit_config/package.xml':
Traceback (most recent call last):
  File "/opt/ros/humble/share/ament_cmake_core/cmake/core/package_xml_2_cmake.py", line 150, in <module>
    main()
  File "/opt/ros/humble/share/ament_cmake_core/cmake/core/package_xml_2_cmake.py", line 53, in main
    raise e
  File "/opt/ros/humble/share/ament_cmake_core/cmake/core/package_xml_2_cmake.py", line 49, in main
    package = parse_package_string(
  File "/home/cfh/.local/lib/python3.10/site-packages/catkin_pkg/package.py", line 788, in parse_package_string
    pkg.validate(warnings=warnings)
  File "/home/cfh/.local/lib/python3.10/site-packages/catkin_pkg/package.py", line 330, in validate
    raise InvalidPackage('\n'.join(errors), self.filename)
catkin_pkg.package.InvalidPackage: Error(s) in package '/home/cfh/ntest/src/six_dof_arm_moveit_config/package.xml':
Invalid email "" for person ""
Maintainers must have an email address
Invalid email "" for person ""
[31mCMake Error at /opt/ros/humble/share/ament_cmake_core/cmake/core/ament_package_xml.cmake:95 (message):
  execute_process(/usr/bin/python3
  /opt/ros/humble/share/ament_cmake_core/cmake/core/package_xml_2_cmake.py
  /home/cfh/ntest/src/six_dof_arm_moveit_config/package.xml
  /home/cfh/ntest/build/six_dof_arm_moveit_config/ament_cmake_core/package.cmake)
  returned error code 1
Call Stack (most recent call first):
  /opt/ros/humble/share/ament_cmake_core/cmake/core/ament_package_xml.cmake:49 (_ament_package_xml)
  /opt/ros/humble/share/ament_lint_auto/cmake/ament_lint_auto_find_test_dependencies.cmake:31 (ament_package_xml)
  CMakeLists.txt:24 (ament_lint_auto_find_test_dependencies)

[0m
-- Configuring incomplete, errors occurred!
See also "/home/cfh/ntest/build/six_dof_arm_moveit_config/CMakeFiles/CMakeOutput.log".
gmake: *** [Makefile:226: cmake_check_build_system] Error 1
